### ISSUE TITLE
Using `make serve` LDAP registration is failing and Rethink is creating scq DB

### ISSUE THREAD

**Issue by wannabeCitizen** [SFII/scq#44]
Using the master branch `make serve` doesn't appear to actually create the db it should and the ldap login fails. Full stack trace below:

```
INFO:tornado.access:304 GET /dashboard (::1) 15.85ms
INFO:tornado.access:200 GET /static/stylesheets/styles.css (::1) 11.99ms
INFO:tornado.access:304 GET /static/dist/all.js (::1) 2.00ms
INFO:tornado.access:200 POST /register/culdap (::1) 10.13ms
ERROR:tornado.application:Uncaught exception POST /register/culdap (::1)
HTTPServerRequest(protocol='http', host='localhost:8000', method='POST', uri='/register/culdap', version='HTTP/1.1', remote_ip='::1', headers={'Referer': 'http://localhost:8000/register/culdap', 'Cookie': '_xsrf=2|d92bcfb7|a2f6a55711fe42011ee5ce5b67ef9e0b|1449213834', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Content-Type': 'application/x-www-form-urlencoded', 'Content-Length': '131', 'Cache-Control': 'max-age=0', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.86 Safari/537.36', 'Upgrade-Insecure-Requests': '1', 'Accept-Language': 'en-US,en;q=0.8', 'Dnt': '1', 'Accept-Encoding': 'gzip, deflate', 'Host': 'localhost:8000', 'Connection': 'keep-alive', 'Origin': 'http://localhost:8000'})
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/tornado/web.py", line 1413, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/Warren/scq/src/handlers/register/culdap_register_handler.py", line 36, in post
    return self.CULdapRegister()
  File "/Users/Warren/scq/src/handlers/register/culdap_register_handler.py", line 96, in CULdapRegister
    user = self.getLdapUser(username, User().REGISTRATION_CULDAP)
  File "/Users/Warren/scq/src/handlers/register_handler.py", line 47, in getLdapUser
    return User().find({'username' : username, 'registration' : registration})
  File "/Users/Warren/scq/src/models/basemodel.py", line 101, in find
    return list(r.db(BaseModel.DB).table(table).filter(key).run(BaseModel.conn))
  File "/Library/Python/2.7/site-packages/rethinkdb/ast.py", line 118, in run
    return c._start(self, **global_optargs)
  File "/Library/Python/2.7/site-packages/rethinkdb/net.py", line 580, in _start
    return self._instance.run_query(q, global_optargs.get('noreply', False))
  File "/Library/Python/2.7/site-packages/rethinkdb/net.py", line 454, in run_query
    raise res.make_error(query)
rethinkdb.errors.ReqlOpFailedError: Database `scq` does not exist in:
r.db('scq').table('User').filter(r.expr({'username': 'misk8072', 'registration': 'registration_culdap'}))
^^^^^^^^^^^
ERROR:tornado.access:500 POST /register/culdap (::1) 3021.10ms
^CTraceback (most recent call last):
  File "server.py", line 49, in <module>
    main()
  File "server.py", line 28, in main
    tornado.ioloop.IOLoop.instance().start()

  File "/usr/local/lib/python3.4/site-packages/tornado/ioloop.py", line 841, in start
    event_pairs = self._impl.poll(poll_timeout)
  File "/usr/local/lib/python3.4/site-packages/tornado/platform/kqueue.py", line 66, in poll
    kevents = self._kqueue.control(None, 1000, timeout)
KeyboardInterrupt
make: *** [serve] Error 1
```

**Comment by untra** [comment]
DB creation was failing silently after we had made some changes to basemodel. #45 addresses the problem and ensures that any other error will properly percolate up.

**Comment by untra** [comment]
Fixed by #45 . closing
